tests/roundtrip.rs

```diff
@@ -0,0 +1,144 @@
+extern crate tempfile;
+extern crate escargot;
+extern crate reqwest;
+extern crate unzip;
+
+use tempfile::tempdir;
+use std::fs::File;
+//use std::path::Path;
+use std::io::{self, Write};
+use std::vec::Vec;
+//use std::env;
+
+#[macro_use]
+extern crate duct;
+
+//use duct::cmd;
+
+#[test]
+fn roundtrip() { //return -> std::io::Result<()> ?
```

Comment:
Yeah, I'd return a Result here. Probably `Result<(), Box<dyn ::std::error::Error>>`, though, as its more generic, or `Result<(), ExitFailure>` which is what `fn main` uses (probably has better error messages).